modules/custom/social_metatag/social_metatag.module

```diff
@@ -34,6 +34,9 @@ function social_metatag_metatag_route_entity(RouteMatchInterface $route_match) {
     'view.group_information.page_group_about',
     'view.group_topics.page_group_topics',
     'view.group_members.page_group_members',
+    'social_group.stream',
+    'view.group_manage_members.page_group_manage_members',
+    'entity.group.content_translation_overview',
```

Comment:
We don't need to add manage members or translation overview here as these are pages that should never be public so even if those pages have OG tags social media can't read them.

Reply:
My idea is always load the preview, because the preview will be the same for each page of the group and the access should be controlled by system permission, because the user can copy any link to share.

If would be better remove private page, let me know